src/pages/AdminLoginPage/index.js

```diff
@@ -99,16 +112,26 @@ class AdminLoginPage extends React.Component {
                   this.handleChange(e);
                 }}
               />
-              <InputText
-                required
-                placeholder="Password"
-                name="password"
-                type="password"
-                value={password}
-                onChange={(e) => {
-                  this.handleChange(e);
-                }}
-              />
+              <div className="password-wrapper">
+                <InputText
+                  required
+                  placeholder="Password"
+                  name="password"
+                  type={passwordShown ? "text" : "password"}
+                  value={password}
+                  onChange={(e) => {
+                    this.handleChange(e);
+                  }}
+                />
+                <div className="CopyIcon">
+                  <CopyText onClick={this.passwordOnClick} />
```

Comment:
What does this do? how is it even necessary, goes to show you copied code without having an understanding of how it works, please try to understand what is required and then implement